frontend/src/entry/model/reviews/ReviewDisplay.tsx

```diff
@@ -11,40 +12,22 @@ export interface ReviewDisplayProps {
 }
 
 export default function ReviewDisplay({ reviewResponses, modelId }: ReviewDisplayProps) {
-  const [acceptedReviewResponses, setAcceptedReviewResponses] = useState<ResponseInterface[]>([])
-  const [changesRequestedReviewResponses, setChangesRequestedReviewResponses] = useState<ResponseInterface[]>([])
-
-  useEffect(() => {
-    const updatedApprovals: ResponseInterface[] = []
-    const updatedRequests: ResponseInterface[] = []
-    reviewResponses.forEach((reviewResponse) => {
-      if (reviewResponse.decision === Decision.Approve) {
-        updatedApprovals.push({ ...reviewResponse, role: reviewResponse.role })
-      }
-      if (reviewResponse.decision === Decision.RequestChanges) {
-        updatedRequests.push({ ...reviewResponse, role: reviewResponse.role })
-      }
-    }),
-      setAcceptedReviewResponses(updatedApprovals)
-    setChangesRequestedReviewResponses(updatedRequests)
-  }, [reviewResponses])
+  const orderedReviewResponses = useMemo(() => reviewResponses.sort(sortByCreatedAtDescending) || [], [reviewResponses])
```

Comment:
As `.sort()` mutates the existing array, I'd suggest using `reviewResponses.toSorted(sortByCreatedAtDescending)`. This works the same as `.sort()` but leaves the original array untouched which will avoid any unwanted behaviour in parent components.

Reply:
Didn't know about `toSorted` interesting!